ocs_ci/ocs/node.py

```diff
@@ -2667,3 +2667,119 @@ def generate_nodes_for_provider_worker_node_tests():
     generated_nodes = get_node_objs(node_choice_names)
     log.info(f"Generated nodes for provider node tests: {node_choice_names}")
     return generated_nodes
+
+
+def gracefully_reboot_nodes():
+    """
+
+    Gracefully reboot OpenShift Container Platform nodes
+
+    """
+    from ocs_ci.ocs import platform_nodes
+
+    node_objs = get_node_objs()
+    factory = platform_nodes.PlatformNodesFactory()
+    nodes = factory.get_nodes_platform()
+    waiting_time = 30
+    for node in node_objs:
+        node_name = node.name
+        unschedule_nodes([node_name])
+        drain_nodes([node_name])
+        nodes.restart_nodes([node], wait=False)
+        log.info(f"Waiting for {waiting_time} seconds")
+        time.sleep(waiting_time)
```

Comment:
can't we use a function to wait for node connectivity instead?

Reply:
Yes, it makes sense. But this function was initially created by Akarsha Rai: https://github.com/red-hat-storage/ocs-ci/blame/master/ocs_ci/ocs/node.py#L2722. I needed to rebase from the `release-4.11`. That's why the change is included in the current PR. I can make another PR from the `master` branch and change the above code.